docs/execution-providers/OpenVINO-ExecutionProvider.md

```diff
@@ -50,15 +50,12 @@ pip install onnxruntime-openvino
 
 * **Windows**
 
-   To enable OpenVINO™ Execution Provider with ONNX Runtime on Windows we must install OpenVINO™ separately:
-      ```
-      pip install openvino
-      ```
-   Code to be added in ONNX Runtime Windows Samples:
-      ```
-      import openvino.utils as utils
-      utils.add_openvino_libs_to_path()
-      ```
+   To enable OpenVINO™ Execution Provider with ONNX Runtime on Windows it is must to set up the OpenVINO™ Environment Variables using the full installer package of OpenVINO™.
```

Comment:
Should the above install instructions be left in the document instead of deleted ie does the user still need to install OpenVINO?

Reply:
The user doesn't need to install openvino through pip and use it in this way because `utils.add_openvino_libs_to_path()` is openvino internal function and doesn't work since 2022.3 and will disappear in 2023.0.